Libraries/Image/ImageProps.js

```diff
@@ -14,6 +14,7 @@ import type {SyntheticEvent, LayoutEvent} from '../Types/CoreEventTypes';
 import type {EdgeInsetsProp} from '../StyleSheet/EdgeInsetsPropType';
 import type {ImageSource} from './ImageSource';
 import type {ViewStyleProp, ImageStyleProp} from '../StyleSheet/StyleSheet';
+import type {____ColorValue_Internal} from '../StyleSheet/StyleSheetTypes';
```

Comment:
I think we'll want to import the type like this: https://github.com/facebook/react-native/blob/d7921f05048c9163d7d2c102e5226abf59d88948/Libraries/StyleSheet/PlatformColorValueTypes.ios.js#L11

Reply:
Good point, I've just updated it.